src/explore-education-statistics-frontend/src/components/Pagination.tsx

```diff
@@ -0,0 +1,11 @@
+import BasePagination, { PaginationProps } from '@common/components/Pagination';
+import Link from '@frontend/components/Link';
+import React from 'react';
+
+const Pagination = ({ ...props }: Omit<PaginationProps, 'renderLink'>) => (
+  <BasePagination
+    {...props}
+    renderLink={({ ...linkProps }) => <Link {...linkProps} />}
+  />
```

Comment:
Would suggest integrating Next's router into the `baseUrl` and `queryParams` props as defaults:

```tsx
const router = useRouter();

<BasePagination
  baseUrl={baseUrl ?? router.pathname}
  queryParams={queryParams ?? router.query}
  renderLink={({ ...linkProps }) => <Link {...linkProps} />}
/>